Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "malachite-bigint"
-version = "0.1.8"
+version = "0.1.0"
 authors = ["Steve Shi <shikangzhi@gmail.com>"]
 edition = "2021"
 license = "MIT"
```

Comment:
Could we add a bit of clarification on the license? Currently, this repo doesn't have a LICENSE document, and this license (MIT/X11) is different from `malachite`'s license (LGPL-3.0).

Reply:
I vote for following malachite license
@qingshi163 do you have preferences?

Reply:
Following malachite license make sense for me